tests/gpu/FixedFunctionOp_test.cpp

```diff
@@ -173,6 +173,136 @@ OCIO_ADD_GPU_TEST(FixedFunction, style_aces_glow10_inv)
     test.setErrorThreshold(1e-7f);
 }
 
+OCIO_ADD_GPU_TEST(FixedFunction, style_aces_gamutmap13_fwd)
+{
+    const double data[7] = { 1.147, 1.264, 1.312, 0.815, 0.803, 0.880, 1.2 };
+    OCIO::FixedFunctionTransformRcPtr func =
+        OCIO::FixedFunctionTransform::Create(OCIO::FIXED_FUNCTION_ACES_GAMUTMAP_13, &data[0], 7);
+    func->setDirection(OCIO::TRANSFORM_DIR_FORWARD);
+
+    test.setProcessor(func);
+
+    OCIOGPUTest::CustomValues values;
+    values.m_inputValues =
```

Comment:
Again, this is awesome but would be nice to put a comment about the source of the data.

Reply:
Added a mention pointing to `FixedFunctionOpCPU_tests.cpp`.